spacy_llm/tests/pipeline/test_llm.py

```diff
@@ -72,7 +78,7 @@ def test_type_checking_valid() -> None:
         nlp.add_pipe("llm", config={"task": {"@llm_tasks": "spacy.NoOp.v1"}})
```

Comment:
Should we be using the fixture in here?

Reply:
No, we deliberately test the default config here. The fixture is for a NoOp backend setup.

Reply:
Sounds good. Then we don't need to pass the fixture though, but I think we're fine as it is.